Gruntfile.js

```diff
@@ -206,37 +242,13 @@ module.exports = (grunt) => {
       files: ['tests/flow-headless-browser.html'],
     },
     copy: {
+      // The build/esm/ folder needs a package.json that says "type": "module".
       modulePackageJSON: {
         expand: true,
         cwd: BASE_DIR,
         src: ['tools/esm/package.json'],
-        dest: path.join(BUILD_DIR, 'esm/'),
-        flatten: true,
-      },
-      // Modules: For now, we just copy the vexflow****.js to the esm/ folder and add exports at the bottom.
-      moduleJSFiles: {
-        expand: true,
-        cwd: BUILD_DIR,
-        src: ['*.js'],
-        dest: path.join(BUILD_DIR, 'esm/'),
+        dest: BUILD_ESM_DIR,
         flatten: true,
-        options: {
-          process: (content) => {
-            // We take a JS file that sets globalThis.Vex, and turn it into a ES module.
-            // To do this, we extract all the classes from Vex.Flow, and re-export them.
-            // We also export Vex both as a named export and as a default export.
-            // THIS IS A HACK, so we should figure out how to use webpack to export both ESM and CJS.
-            const exports = `
-const Vex = globalThis['Vex'];
-const Flow = Vex.Flow;
-const  { Accidental, Annotation, Articulation, BarNote, Beam, Bend, BoundingBox, BoundingBoxComputation, ChordSymbol, Clef, ClefNote, Crescendo, Curve, Dot, EasyScore, Element, Factory, Font, Formatter, Fraction, FretHandFinger, GhostNote, Glyph, GlyphNote, GraceNote, GraceNoteGroup, GraceTabNote, KeyManager, KeySignature, KeySigNote, Modifier, ModifierContext, MultiMeasureRest, Music, Note, NoteHead, NoteSubGroup, Ornament, Parser, PedalMarking, Registry, RenderContext, Renderer, RepeatNote, Stave, Barline, StaveConnector, StaveHairpin, StaveLine, StaveModifier, StaveNote, Repetition, StaveTempo, StaveText, StaveTie, Volta, Stem, StringNumber, Stroke, System, Tables, TabNote, TabSlide, TabStave, TabTie, TextBracket, TextDynamics, TextFormatter, TextNote, TickContext, TimeSignature, TimeSigNote, Tremolo, Tuning, Tuplet, Vibrato, VibratoBracket, Voice, BarlineType, ModifierPosition } = Vex.Flow;
```

Comment:
HACK removed!

We build the ESM output using the tsc compiler, which basically strips all the type information from the TypeScript files. If you look at the build/esm/ folder, you'll see that ESM is basically TS without static types.